tests/cdc-wal2json/000000010000000000000002.sql

```diff
@@ -55,6 +55,14 @@ EXECUTE dc973d3c["11.990000","28814","592","1","3973","11.950000","2022-07-06 12
 PREPARE dc973d3c AS UPDATE "public"."payment_p2022_07" SET "amount" = $1 WHERE "payment_id" = $2 and "customer_id" = $3 and "staff_id" = $4 and "rental_id" = $5 and "amount" = $6 and "payment_date" = $7;
 EXECUTE dc973d3c["11.990000","29136","13","2","8831","11.950000","2022-07-22 16:15:40.797771+00"];
 COMMIT; -- {"xid":501,"lsn":"0/24E7158","timestamp":"2024-02-02 09:33:28.573357+0000"}
+BEGIN; -- {"xid":502,"lsn":"0/24E3448","timestamp":"2024-04-24 07:27:57.986851+0000","commit_lsn":"0/24E34F8"}
+PREPARE c351164d AS DELETE FROM "public"."address" WHERE "address_id" = $1;
+EXECUTE c351164d["1"];
+PREPARE c351164d AS DELETE FROM "public"."address" WHERE "address_id" = $1;
+EXECUTE c351164d["3"];
+PREPARE 575c8f00 AS UPDATE "public"."address" SET "address" = $1, "address2" = $2, "district" = $3, "city_id" = $4, "postal_code" = $5, "phone" = $6, "last_update" = $7 WHERE "address_id" = $8;
+EXECUTE 575c8f00["1411 Lillydale Drive",null,"QLD","576","751007","6172235589","2022-02-15 09:45:30+00","4"];
+COMMIT; -- {"xid":502,"lsn":"0/24E34F8","timestamp":"2024-04-24 07:27:57.986851+0000"}
 BEGIN; -- {"xid":502,"lsn":"0/24E7158","timestamp":"2024-02-02 09:33:28.573418+0000","commit_lsn":"0/24E7800"}
```

Comment:
I did not update the `xid` and `timestamp` in other statements since they are ignored during comparison. `copy.sh` only displays the changes, making it easier to add the missing statements alone. I hope this is fine.